docs/DeveloperGuide.md

```diff
@@ -230,6 +230,75 @@ parses the user's input as a file name and then creates a `SnapCommand` object w
 * We chose to prevent users from creating a snapshot if the specified file name already exists
 as overwriting a file is irreversible and would be disastrous for zookeepers if done unintentionally
 
+### Feeding times feature (by Jeremy)
+
+#### Implementation
+
+The feeding time feature utilizes a TreeSet with a custom comparator.
+
+Each Animal object has a `FeedTimes` TreeSet.
+
+The custom comparator `FeedTimeComparator` compares the integer values of the feeding times, returning them in ascending order.
+
+The feeding times feature allows for the following functionallity:
+
+* Add multiple feeding times to each animal listing.
+* Ensure feeding times are always displayed in chronological order.
+
+The following notable methods are used for the feeding times feature:
+* `parseFeedTimes` (in `ParserUtil` class) - returns a Set of `FeedTime` objects from user input
```

Comment:
For consistency, name methods in form Class#method(parameter). e.g. ParserUtil#parseFeedTimes()

Reply:
Updated to match format